source/clear-linux/reference/collaboration/collaboration.rst

```diff
@@ -0,0 +1,17 @@
+.. _collaboration:
+
+Collaboration guidelines
+########################
+
+Thank you for your interest in collaborating with the |CLOSIA|. This guide
+details the best ways to communicate and collaborate with the |CL| team. The
+documentation section provides all the information you need to help us
+improve our documents with your use case tutorials, examples, or task focused
+guides. Read the information carefully before submitting any work for review
+to ensure your contribution can be added as quickly as possible.
```

Comment:
There are many ways to reach out to us but the paths detailed here help ensure ...